.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  build-and-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Generate changelog
```

Comment:
This should happen in the existing bump-version workflow already. Maybe it makes sense to combine the 2 and rename it to create-new-release-version or something?

Reply:
Done